pyorthanc/retrieve.py

```diff
@@ -0,0 +1,93 @@
+import os
+from typing import List
+
+from .instance import Instance
+from .patient import Patient
+from .series import Series
+from .study import Study
+
+
+def retrieve_and_write_patients(patients: List[Patient], path: str) -> None:
+    """Retrieve and write patients to given path
+
+    Parameters
+    ----------
+    patients
+        List of patients.
+    path
+        Path where you want to write the files.
+    """
+    os.makedirs(path, exist_ok=True)
```

Comment:
It was because of the weird things that were done when writing files (it used to check all the files in the directory to make the pathname, so that we get CT-1, CT-2, etc.). Now I use the "PatientID/StudyInstanceUID/SeriesInstanceUID/SOPInstanceUID.dcm", which is much more simple.

Users who want to control the pathname can write their own `retrieve_and_write` function since it is quite simple.

It looks OK to you?